http-filter-example/http_filter_config.cc

```diff
@@ -14,18 +14,18 @@ namespace Configuration {
 
 class HttpSampleDecoderFilterConfig : public NamedHttpFilterConfigFactory {
 public:
-  HttpFilterFactoryCb createFilterFactory(const Json::Object& json_config, const std::string&,
-                                          FactoryContext& context) override {
+  Http::HttpFilterFactoryCb createFilterFactory(const Json::Object& json_config, const std::string&,
```

Comment:
nit: Can we also rename this `Http::FilterFactoryCb` as long as we are doing this?